File: C++/1347. Minimum Number of Steps to Make Two Strings Anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        vector<int> v1(26);
        vector<int> v2(26);
        for(char i: s){
            v1[i-'a']++;
        }
        for(char i: t){
            v2[i-'a']++;
        }
        int res = 0;
        for(int i=0; i<26; i++){
            res+=abs(v1[i]-v2[i]);
        }
        return res/2;
    }
};

File: C#/1347. Minimum Number of Steps to Make Two Strings Anagram.cs
public class Solution {
    public int MinSteps(string s, string t) {
        int[] arr1 = new int[26];
        int[] arr2 = new int[26];
        foreach(char i in s){
            arr1[i-'a']++;
        }
        foreach(char i in t){
            arr2[i-'a']++;
        }
        int res = 0;
        for(int i=0; i<26; i++){
            res+=Math.Abs(arr1[i]-arr2[i]);
        }
        return res/2;
    }
}
